.gitignore

```diff
@@ -5,4 +5,4 @@ erl_crash.dump
 production_cluster
 noisepagecontrol/primary_worker/workload_captures/*
 noisepagecontrol/control_plane/resources/*
-noisepagecontrol/logs/*
```

Comment:
please add this back

Reply:
don't have to add this back as the rule is covered in noisepagecontrol/logs/.gitignore